itou/templates/apply/submit_step_pending_authorization.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layout/content.html" %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-12 col-lg-7 pl-lg-5">
+            <div class="card c-card mb-3 mb-lg-5 p-3">
+                <div class="card-header">
+                    <h1 class="h2">Status de prescripteur habilité non vérifié</h1>
+                </div>
+                <div class="card-body">
+                    <div class="alert alert-warning" role="status">
+                        <div class="row">
+                            <div class="col-auto pr-0">
+                                <i class="ri-information-line ri-xl text-warning"></i>
+                            </div>
+                            <div class="col">
+                                <p class="mb-2"><strong>Attention, si vous souhaitez orienter un candidat auprès d'une SIAE, vous ne pourrez pas valider son éligibilité à l'IAE.</strong></p>
+                                <p class="mb-0">Votre statut de prescripteur habilité n'a pas encore été vérifié par nos équipes. Vous devriez être notifié(e)s de sa confirmation dans un délai de 72h.</p>
```

Comment:
Je suis pas sûr pour le pluriel là

```suggestion
                                <p class="mb-0">Votre statut de prescripteur habilité n'a pas encore été vérifié par nos équipes. Vous devriez être notifié(e) de sa confirmation dans un délai de 72h.</p>
```